### PR TITLE
Worksheet submission - Oskar Korpysa

### PR DESCRIPTION
**Pull request by ghost**
added the game

### CODE REVIEW

**Comment by ghost** [comment]
Oskar Korpysa submission

**Comment by Oskey2** [comment]
Worksheet submission - Oskar Korpysa

File: comp110-worksheet-6/OxoBoard.cs [hunk 1]
```diff
@@ -6,71 +6,130 @@
...
+        public Mark GetWinner()
+        {
+            for (int i = 0; i < board.GetLength(0); i++)
+            {
+                for (int j = 0; j < board.GetLength(1); j++)
+                {
+                    if (board[i, j] != Mark.None) //Only look inside the cell if it's not empty.
+                    {
+                        int currI = i + 2, currJ = j + 2;
+                        int[] currSquare = new int[] { i, j };
```
Comment (edpowley) [inline comp110-worksheet-6/OxoBoard.cs:79]: Are these used anywhere?

File: comp110-worksheet-6/OxoBoard.cs [hunk 2]
```diff
@@ -6,71 +6,130 @@
...
+                        if (currI < 3 && currJ < 3)
```
Comment (edpowley) [inline comp110-worksheet-6/OxoBoard.cs:80]: What are you intending to check here? I think this is just causing some checks to be skipped, hence the failed unit tests

**Pull request closed without merging**